src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -14,7 +14,9 @@
   <ItemGroup>
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\Utils.cs" />
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.cs" />
+    <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.CoreCLR.cs" />
```

Comment:
I think we better use .NetCore.cs instead of .CoreCLR.cs for .NET Core specific tests.